packages/ic-management/src/utils/ic-management.utils.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { uint8ArrayToHexString } from "@dfinity/utils";
+import { mockCanisterId } from "../ic-management.mock";
+import { decodeSnapshotId, encodeSnapshotId } from "./ic-management.utils";
+
+describe("ic-management.utils", () => {
+  const mockLocalSubnetId = [0, 0, 0, 0, 0, 0, 0, 1];
+
+  const mockSnapshotId = Uint8Array.from([
+    ...mockCanisterId.toUint8Array(),
+    ...mockLocalSubnetId,
+  ]);
+
+  it("should correctly encode a snapshot ID with encodeSnapshotId", () => {
+    const expectedHex = uint8ArrayToHexString(mockSnapshotId);
```

Comment:
This way the test just has the same implementation as the code under test. If that implementation is wrong, you don't catch anything.
Let's just hard code some values instead.

Reply:
Sure, done.